src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/DefaultGATKReadFilterArgumentCollection.java

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.hellbender.cmdline.GATKPlugin;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+
+import java.util.*;
+
+/**
+ * {@link GATKReadFilterArgumentCollection} for optional read filters in the command line. It allows:
```

Comment:
Rephrased to: `Default {@link GATKReadFilterArgumentCollection} applied in GATK for optional read filters in the command line. It contains arguments that allow the user to:`